addons/osfstorage/models.py

```diff
@@ -4,6 +4,7 @@
 
 from django.apps import apps
 from django.db import models, connection
+from django.core.cache import cache
```

Comment:
You'll need to configure the caching backend, so that cache results get stored in the DB. See the docs: https://docs.djangoproject.com/en/1.11/topics/cache/